lib/index.js

```diff
@@ -20,15 +22,30 @@ const LICENSES = [
   'licence.md'
 ]
 
+const noop = () => {}
+
 module.exports = getLicense
 function getLicense (modulePath, optionalCallback) {
+  const cb = optionalCallback || noop
+
   return new Promise((resolve, reject) => {
-    const pkg = require(path.join(modulePath, 'package.json'))
+    let pkg = {}
+
+    try {
```

Comment:
This is done in order to forward the error with the callback interface as well. If we don't do this the callback will never get executed when the user typed in the wrong module path.